upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -374,6 +374,17 @@ There you can find links to upgrade notes for other versions too.
 - improve your data fixtures and tests so they are more resistant against domains and locales settings changes [#1425](https://github.com/shopsys/shopsys/pull/1425)
     - if you have done a lot of changes in your data fixtures you might consider to skip this upgrade
     - for detailed information, see [the separate article](upgrade-instructions-for-improved-data-fixtures-and-tests.md)
+- fix your acceptance tests for single domain: ([#1477](https://github.com/shopsys/shopsys/pull/1477))
+    - create second currency only for multidomain project ([diff](https://github.com/shopsys/shopsys/pull/1477/files))
+    - fix used translation message ID in `PaymentImageUploadCest`
+    ```diff
+    -    $me->seeTranslationAdmin('Payment <strong><a href="{{ url }}">%name%</a></strong> modified', 'messages', [
+    +    $me->seeTranslationAdmin('Payment <strong><a href="{{ url }}">{{ name }}</a></strong> modified', 'messages', [
+             '{{ url }}' => '',
+    -        '%name%' => t('Credit card', [], 'dataFixtures', $me->getAdminLocale()),
+    +        '{{ name }}' => t('Credit card', [], 'dataFixtures', $me->getAdminLocale()),
+         ]);
+    ```
```

Comment:
Can you integrate this upgrade instruction to `upgrade-instructions-for-improved-data-fixtures-and-tests.md`?
I believe it was introduced in reworked datafixtures (and they were not yet released) and imagine someone decides to skip the improve data fixtures step and next instruction is not valid for him and could be confusing

Reply:
Done